src/FirePokemon.java

```diff
@@ -0,0 +1,41 @@
+package src;
+
+public class FirePokemon extends Pokémon {
+    private int flameThrower;
+    private String fireLash;
+
+    public FirePokemon(String sound, String food, int flameThrower, String fireLash) {
+        super(sound, food);
+        this.flameThrower = flameThrower;
+        this.fireLash = fireLash;
+    }
+
+    public int getFlameThrower() {
+        return flameThrower;
+    }
+
+    public String getFireLash() {
+        return fireLash;
+    }
+
+    public void setFlameThrower(int flameThrower) {
+        this.flameThrower = flameThrower;
+    }
+
+    public void setFireLash(String fireLash) {
+        this.fireLash = fireLash;
+    }
+
+    public String nextLevel() {
```

Comment:
Deze methode heb je in elke subPokemon gemaakt en op dezelfde manier. Misschien dat deze methode dan beter in de superPokemon klasse kan staan. 
Misschien dat je inplaats van een String returnen, een `int level` attribuut kunt maken en die +1 doen. 
In je `level` methode kun je dan iets als `"current level is" + level` printen. 
Nu geeft deze methode altijd dezelfde uitkomst, ongeacht wat. Eigenlijk is het dus een constante en kun je net zo goed bovenaan je Pokemon klasse `String level = "level 2"` zetten of zelfs `private final String level = "level 2"`